frontend/packages/noobaa-storage-plugin/integration-tests/tests/overviewDashboard.scenario.ts

```diff
@@ -0,0 +1,106 @@
+import { execSync } from 'child_process';
```

Comment:
The file should be named to something like `object-service-dashboard` instead of `overviewDashboard`